examples/morpheusvm/tests/integration/integration_test.go

```diff
@@ -213,7 +213,8 @@ var _ = ginkgo.BeforeSuite(func() {
 		toEngine := make(chan common.Message, 1)
 		db := memdb.New()
 
-		v := controller.New()
+		v, err := controller.New(vm.WithManualGossiper(), vm.WithManualBuilder())
```

Comment:
non-commitable suggestion, but could we make this:

```suggestion
		v, err := controller.New(vm.WithManual())
```

Just a convenience function for testing